src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous" >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__container">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Product">
+
+        <h1 class="card__name">APPLE A1419 iMac 27" Retina  5K Monoblock (MNED2UA/A)</h1>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars__stars--4">
+            <span class="stars__item"></span>
+            <span class="stars__item"></span>
+            <span class="stars__item"></span>
+            <span class="stars__item"></span>
+            <span class="stars__item"></span>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <h2 class="card__title">Price:</h2>
+          <p class="card__cost">$2,199</p>
+        </div>
+
+        <a
+          class="card__item"
+          href="#link"
+          data-qa="hover"
+          >
+          <span class="card__link">Buy</span>
+        </a>
```

Comment:
Probably no need to have an additional span here
It is not a problem in itself, but you have two classes (`card__item` and `card__link`) instead of having one.